export const authenticate= async (req,res,next)=>{
    try {
        
    } catch (error) {
        next(error)
    }
}